extendeddatacrossword.go

```diff
@@ -345,8 +363,8 @@ func noMissingData(input [][]byte) bool {
 	return true
 }
 
-func (eds *ExtendedDataSquare) computeSharesRoot(shares [][]byte, i uint) []byte {
-	tree := eds.createTreeFn()
+func (eds *ExtendedDataSquare) computeSharesRoot(shares [][]byte, axis Axis, i uint) []byte {
+	tree := eds.createTreeFn(axis, i)
 	for cell, d := range shares {
 		tree.Push(d, SquareIndex{Cell: uint(cell), Axis: i})
```

Comment:
I might be reading this a bit wrong, but isn't the dual axis/Axis terminology a bit confusing?

Reply:
It's more descriptive than calling the variable something like `a` imo